### PR TITLE
PORTAL-5722 Rewrite Lgputil as code based for Vputil

### PR DESCRIPTION
**Pull request by markkong318**
# Change
- Set `uri` in `app.json`
- Add reset leaderbaord and test
- Fix apollo import error when used in vputil level
- Fix test error in node 20
- Mask auth info in log

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
   "name": "lgputil",
-  "version": "1.4.4",
+  "version": "1.4.5",
   "description": "CLI utility for Link Game Platform",
   "main": "src/index.js",
   "type": "module",
   "scripts": {
     "test": "cd test && node --loader=esmock ../node_modules/jasmine/bin/jasmine.js",
-    "bats": "GAME_ID=`sed -n 's/^.*\"GAME_ID=\\([^\"]*\\).*/\\1/p' ./.github/workflows/suites.yml` ./test/bats/suites.bats"
+    "bats": "CMD=`grep '\"name\"' package.json | awk -F '\"' '{print $4}'` GAME_ID=`sed -n 's/^.*\"GAME_ID=\\([^\"]*\\).*/\\1/p' ./.github/workflows/suites.yml` ./test/bats/suites.bats"
```
Comment (toruta39) [inline package.json:9]: why not just lgputil

**Review by toruta39 — COMMENTED** [review]
mostly fine, one question

File: package.json (hunk 1 above)
Comment (markkong318) [reply package.json:9]: just want to let the CMD be filled in automatically. but I think it may not need now. recover it

Comment (markkong318) [reply package.json]: recovered

**Review by toruta39 — APPROVED** [review]
LGTM

**Pull request merged**